sale_report_salesperson_from_partner/tests/test_sale_report_salesperson_from_partner.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, common, new_test_user
+
+
+class SaleReportSalespersonFromPartner(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Remove this variable in v16 and put instead:
+        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+        DISABLED_MAIL_CONTEXT = {
+            "tracking_disable": True,
+            "mail_create_nolog": True,
+            "mail_create_nosubscribe": True,
+            "mail_notrack": True,
+            "no_reset_password": True,
+        }
+        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
+        cls.salesperson = new_test_user(
```

Comment:
Could you inherit from [BaseCommon](https://github.com/odoo/odoo/blob/8f0b8d451f97dd7d0f118b541a80e9674af51124/odoo/addons/base/tests/common.py#L19)

Reply:
Right, thanks